text/0056-tx-fee-map-digest.md

```diff
@@ -0,0 +1,95 @@
+- Feature Name: tx-fee-map-digest
+- Start Date: 2022-11-03
+- MCIP PR: https://github.com/mobilecoinfoundation/mobilecoin/pull/2793
+- Tracking Issue: https://github.com/mobilecoinfoundation/mobilecoin/issues/2611
+
+# Summary
+[summary]: #summary
+
+Modify the `Tx` struct to include a digest of the minimum fee map, which provides a mitigation against a possible information leak attack that can be executed by a malicious node operator.
+
+A security audit performed by Trail of Bits has identified a potential information leak attack: The consensus enclave fee map can be changed by the untrusted side, by restarting the node. If the attacker controls a node and is able to get the user to submit their Tx once with one fee map, and once with another, then the priority value leaks information about what the token id of the Tx is.
+
+In principle they could do this without the user knowing -- they would restart the node and it would not be able to attest to the peers after changing the fee map, but it could still accept Tx proposals and see the WellFormedTxContext which conatins the priority number. Then they turn the node back to normal. The user's app retries when their transaction doesn't land, and the same Tx comes back to their node, now with the regular fee map.
```

Comment:
:question: Is "WellFormedTxContext" meant to be a code type or similar, seems odd to have no spaces and no markup
```suggestion
In principle they could do this without the user knowing -- they would restart the node and it would not be able to attest to the peers after changing the fee map, but it could still accept Tx proposals and see the `WellFormedTxContext` which contains the priority number. Then they turn the node back to normal. The user's app retries when their transaction doesn't land, and the same Tx comes back to their node, now with the regular fee map.
```

Reply:
yeah, it's code